console/src/routes/events/+page.svelte

```diff
@@ -5,45 +5,32 @@
 	import type { PageData } from './$types';
 	import Loading from '$lib/components/Loading.svelte';
 	import type { EventTable } from '$lib/events/api';
-
-	//Set default tab
-	let tab = 'Events';
+	import { goto } from '$app/navigation';
+	import HelperText from '$lib/components/HelperText.svelte';
 
 	export let data: PageData;
 
 	// Event logs table data
 	$: tableData = data.tableData;
 	$: ago = data.ago;
 	$: loading = data.tableData === undefined;
-
-	// Dummy Pipeline data
-	let pipelineData = [
-		{
-			name: 'Pipeline Name #1',
-			metadata: 'Metadata',
-			pastStatus: ['success', 'failure', 'success', 'success', 'failure']
-		},
-		{
-			name: 'Pipeline Name #2',
-			metadata: 'Metadata',
-			pastStatus: ['failure', 'failure', 'failure', 'success', 'failure']
-		},
-		{
-			name: 'Pipeline Name #3',
-			metadata: 'Metadata',
-			pastStatus: ['success', 'failure', 'failure', 'success', 'failure']
-		}
-	];
+	$: tab = data.tab;
 
 	$: activeRow = tableData?.[0];
 
 	function setActiveRow(row: EventTable) {
 		activeRow = row;
 	}
+
+	const sourceHelperText =
+		'<strong>Recent source events:</strong> Generated by apps, tasks and schedules when a pipeline runs';
+
+	const allHelperText =
+		'<strong>All recent events:</strong> Includes source events + other events that occur when a source event is generated';
```

Comment:
The other events don't occur when a source event is generated. They occur when a pipeline runs.

Also has the open `<strong>` tag and no closing one

Reply:
Updated the helper text.
The `</strong>` closing tag is already present after 'All recent events:'